pkg/config/systemd.go

```diff
@@ -67,12 +67,20 @@ func useJournald() bool {
 		if !useSystemd() {
 			return
 		}
-		journal, err := sdjournal.NewJournal()
-		if err != nil {
-			return
+		for _, root := range []string{"/run/log/journal", "/var/log/journal"} {
```

Comment:
Should this also handle entry from `.conf` since journal paths can be custom as well afaik.

Reply:
We can do that as a followon.